chicago/bills.py

```diff
@@ -97,11 +96,15 @@ def scrape(self):
                                                            'communication', 
                                                            'report', 
                                                            'oath of office') :
-                    continue
+                    bill_type = None
                 else :
                     bill_type = legislation_summary['Type'].lower()
 
-                bill_session = self.session(legislation_summary['Intro\xa0Date'])
+                try :
+                    bill_session = self.session(self.toTime(legislation_summary['Intro\xa0Date']))
+                except :
+                    print(legislation_summary)
```

Comment:
It turns out there's a lot of interesting stuff you can do with exceptions in python3 that I just learned from a convo with @paultag so I'm going to let him chime in if he's interested